src/Symfony/Component/EventDispatcher/EventDispatcher.php

```diff
@@ -80,7 +80,7 @@ public function getListeners($eventName = null)
      */
     public function hasListeners($eventName = null)
     {
-        return (bool) count($this->getListeners($eventName));
+        return (bool) $this->getListeners($eventName);
```

Comment:
count works on everything that implements `\Countable`, so if it returns an object while the method name implies a collection of some sorts, I expect it to also be countable.